ngraph/test/frontend/mock_frontend.cpp

```diff
@@ -29,6 +29,6 @@ extern "C" MOCK_API void* GetFrontEndData()
 {
     FrontEndPluginInfo* res = new FrontEndPluginInfo();
     res->m_name = "mock1";
-    res->m_creator = [](FrontEndCapFlags) { return std::make_shared<FrontEndMock>(); };
```

Comment:
@slyalin I see that `FrontEndCapFlags` are removed, does it mean we can merge ONNX editor and importer as it as before?

Reply:
It is more complicated. By removing caps we just allow FE designers to organize libraries in their own way and load them on-demand. I don't know what the benefit of division to two libraries right now, because I don't know binary size impact for each schema. ONNX team should present input data to make a decision.